apps/member-profile/app/routes/_profile.companies_.$id.tsx

```diff
@@ -154,6 +155,10 @@ export default function CompanyPage() {
             {company.levelsFyiSlug && (
               <LevelsFyiLink slug={company.levelsFyiSlug} />
             )}
+
+            {company.leetcodeTagged && (
+              <LeetcodeTaggedLink leetcode_tagged={company.leetcodeTagged} />
```

Comment:
let's change the naming from `leetcode_tagged` to `slug`, so we can have consistency with the pattern used for the Levels.fyi link as well